src/lightning/pytorch/loggers/neptune.py

```diff
@@ -43,6 +43,15 @@
 _INTEGRATION_VERSION_KEY = "source_code/integrations/pytorch-lightning"
 
 
+def _get_expected_model_path(dir_path: str) -> str:
```

Comment:
can we add simple doctest to better illustrate what it does

Reply:
Sure :smile: